commisery/versioning.py

```diff
@@ -0,0 +1,463 @@
+# Copyright (c) 2018 - 2022 TomTom N.V. (https://tomtom.com)
```

Comment:
Current policy accepts only indicating the starting year (simplifying maintenance going forward)

Reply:
I chose not to do that form and only bump the modification year, as the rest of the repo uses that form.